mDNSWindows/Poll.c

```diff
@@ -113,7 +113,7 @@ ShiftDown( void * arr, size_t arraySize, size_t itemSize, int index )
 
 #define	DEBUG_NAME	"[mDNSWin32] "
 #define gMDNSRecord mDNSStorage
-mDNSlocal Poll gPoll = { mDNSfalse, NULL };
+mDNSlocal Poll gPoll = {};
```

Comment:
Why is this change necessary?

Reply:
Because it initializes only the first two fields of `struct Poll_struct` / `Poll`. The changed line initializes the whole struct. I don't remember the exact warning text, but I can check it for you if you're curious.

Reply:
Ah, okay, that explains it. I don't see any issue with this—just wanted a reason to mention. :)